### ISSUE TITLE
Glitchy item bag and PC

### ISSUE THREAD

**Issue by Jastolze** [TheFakeMateo/rpp-backup#8]
I compiled a custom version of Red Plus Plus and used the hard version (pokeblue). Everything was going fine until I killed a wild Steelix in Digletts Cave (I changed encounters throughout the game) towards the end of the battle, it got gltichy right after I was gaining EXP. 

The game then resumed to normal and I didn't notice anything for a while, but I saw I had an item in my bag called 'Headbutt' and when I selected it, it threw a pokeball (I think it was a Great Ball). Other weird things happened after that battle, like my PC having a bunch of blank spaces after a certain item number and then in my bag, one of the items was made to have like 128 of it. I think it was the 8th item slot. (It looked like the gltich after you encounter Missingno and it changes your 6th item slot to a number above 99.

Also on a previous version, (about a year ago) I was playing this on my phone and I was in Diglett Cave. Anyway, as before I changed the encounters in the cave. Weirdly enough, occasionally I would run into a Lv0 Caterpie and if I did anything except run away, the game would lock. I checked the RPP source to make sure there wasn't a Caterpie and there wasn't.

I like this game a lot, but would rather play if it's not going to do those things again. Does my changing the encounter rate have something to do with this? If you'd like, I can send a screenshot of the issue, and my rom/save file. I was originally playing on my phone and transferred it to 3DS VC.

**Comment by JustRegularLuna** [comment]
It might be more helpful to show me what you did when you were trying to change the encounter rate. Just changing it shouldn't hurt anything, but if you did something incorrectly then it might have.

**Comment by Jastolze** [comment]
I've come to the conclusion it could be what device I'm playing it on. I had the issue playing on my GBA flash card (Goomba, which is very slow and outdated) and 3ds VC which isn't really a great device to play rom hacks on.

I could send you the source but just know didn't change much outside of encounter rates, level cap (250) and some move PP/Accuracy.

The same thing happened again a couple days ago while playing it on Goomba, only it showed a shiny pallete, the screen got all weird and I eventually found a way to run. But then the overworld looked corrupted as the battle ended. I had to turn it off and haven't messed with it since.

Weirdly enough, one of the oddities on my 3DS occurred when I ran into a shiny Misdreavus. So maybe the emulator or hardware doesn't handle the shiny pallete correctly. 

Although, you'd probably have a better idea than I would. I'm a very basic rom hacker.
